src/olympia/devhub/templates/devhub/new-landing/components/blog_posts.html

```diff
@@ -0,0 +1,24 @@
+<section class="DevHub-BlogPosts">
+  <h2>
+    <a title="{{ _('Subscribe to this feed') }}" href="http://blog.mozilla.com/addons/feed/">
```

Comment:
Gosh! I fixed that in another place already after I copied the URL from our live environment. Gotta fix that there too :(